packages/contracts-bedrock/src/libraries/Blueprint.sol

```diff
@@ -157,6 +159,45 @@ library Blueprint {
         if (newContract_ == address(0)) revert DeploymentFailed();
     }
 
+    function create(
+        bytes memory _rawBytecode,
+        bytes32 _salt
+    )
+        public
+        returns (address newContract1_, address newContract2_)
+    {
+        if (_rawBytecode.length <= maxInitCodeSize()) {
+            newContract1_ = deploySmallBytecode(blueprintDeployerBytecode(_rawBytecode), _salt);
+            return (newContract1_, address(0));
+        }
+
+        (newContract1_, newContract2_) = deployBigBytecode(_rawBytecode, _salt);
+    }
+
+    function deploySmallBytecode(bytes memory _bytecode, bytes32 _salt) internal returns (address newContract_) {
+        assembly ("memory-safe") {
+            newContract_ := create2(0, add(_bytecode, 0x20), mload(_bytecode), _salt)
+        }
+        require(newContract_ != address(0), "Blueprint: create2 failed");
+    }
+
+    function deployBigBytecode(
+        bytes memory _bytecode,
+        bytes32 _salt
+    )
+        internal
+        returns (address newContract1_, address newContract2_)
+    {
+        uint32 maxSize = maxInitCodeSize();
+        bytes memory part1Slice = Bytes.slice(_bytecode, 0, maxSize);
+        bytes memory part1 = blueprintDeployerBytecode(part1Slice);
+        bytes memory part2Slice = Bytes.slice(_bytecode, maxSize, _bytecode.length - maxSize);
+        bytes memory part2 = blueprintDeployerBytecode(part2Slice);
+
+        newContract1_ = deploySmallBytecode(part1, _salt);
+        newContract2_ = deploySmallBytecode(part2, _salt);
+    }
+
```

Comment:
I like this.

Should add natspec to these methods for consistency with this lib.